tools/aws_lab_setup/teardown_lab.yml

```diff
@@ -7,12 +7,26 @@
   vars:
     ec2_exact_count: 0
     ec2_wait: no
+    teardown: true
+    create_lab: false
 
   roles:
-    - manage_ec2_instances
+     - manage_ec2_instances
```

Comment:
I believe you have an extra space here.

Reply:
good catch!  it still works somehow but I will fix in the next PR